metadata-ingestion/src/datahub/ingestion/source/looker/lookml_source.py

```diff
@@ -669,6 +673,184 @@ def load_viewfile(
         return replace(viewfile, connection=connection)
 
 
+class LookerRefinementResolver:
```

Comment:
Can we extract into refinements.py file?

Reply:
bit hard as this class is depend on data-classes defined in current file